Source/WebKit/Shared/SharedStringHashStore.cpp

```diff
@@ -132,7 +132,7 @@ void SharedStringHashStore::resizeTable(unsigned newTableSize)
     m_tableSize = newTableSize;
 
     if (currentTableMemory) {
-        ASSERT_UNUSED(currentTableSize, currentTableMemory->size() == currentTableSize * sizeof(SharedStringHash));
+        RELEASE_ASSERT(currentTableSize, currentTableMemory->size() == static_cast<size_t>(currentTableSize) * sizeof(SharedStringHash));
```

Comment:
sizeof() returns size_t. decltype(unsigned * size_t) should be size_t. Does this cast produce a behavior change, or is the real fix the RELEASE_ASSERT?

Reply:
We should drop this cast since it is not adding anything. Other part (RELEASE_ASSERT) looks good to me.